frontend/src/components/FundingContent.tsx

```diff
@@ -9,30 +9,29 @@ const FundingContent = () => {
   const { t } = useTranslation("common", { keyPrefix: "Index" });
 
   return (
-    <div className="bg-base-lightest desktop:padding-y-4 tablet:padding-y-2 padding-y-1">
+    <div className="bg-base-lightest padding-y-1 tablet:padding-y-3 desktop-lg:padding-y-6">
       <GridContainer>
-        <Grid row>
-          <h2 className="margin-bottom-0 desktop:font-sans-xl">
-            {t("fo_title")}
-          </h2>
-        </Grid>
-        <Grid row gap="lg">
-          <Grid tablet={{ col: 6 }}>
-            <p className="line-height-sans-4">{t("fo_paragraph_1")}</p>
+        <h2 className="tablet-lg:font-sans-xl">{t("fo_title")}</h2>
+        <Grid row gap>
+          <Grid tabletLg={{ col: 6 }}>
+            <p className="margin-top-0 font-sans-md line-height-sans-4 desktop-lg:line-height-sans-6">
+              {t("fo_paragraph_1")}
+            </p>
           </Grid>
-          <Grid tablet={{ col: 6 }}>
-            <p className="line-height-sans-4">{t("fo_paragraph_2")}</p>
+          <Grid tabletLg={{ col: 6 }}>
```

Comment:
I'm keeping a lot of the content as single-column on smaller screens and multi-column at a higher breakpoints.